contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -268,9 +270,10 @@ GetBulkLoadRequest(StringInfo message)
 			case TDS_TYPE_NVARCHAR:
 			{
 				CheckMessageHasEnoughBytesToRead(&message, STRING_COLUMNMETADATA_LEN);
+				Assert(sizeof(uint16)<=sizeof(colmetadata[currentColumn].maxLen));
```

Comment:
Same here, at multiple locations you have put an assert after CheckMessageHasEnoughBytesToRead, I dont see the point. Please explain.